Source: upstream -> http://redmine.lighttpd.net/issues/2157 
Upstream: yes
Reason: lighttpd won't start with ssl

Index: lighttpd-1.4.26/src/network.c
===================================================================
--- lighttpd-1.4.26/src.orig/network.c
+++ lighttpd-1.4.26/src/network.c
@@ -525,7 +525,7 @@
 
 		if (!s->ssl_use_sslv2) {
 			/* disable SSLv2 */
-			if (SSL_OP_NO_SSLv2 != SSL_CTX_set_options(s->ssl_ctx, SSL_OP_NO_SSLv2)) {
+			if (!(SSL_OP_NO_SSLv2 & SSL_CTX_set_options(s->ssl_ctx, SSL_OP_NO_SSLv2))) {
 				log_error_write(srv, __FILE__, __LINE__, "ss", "SSL:",
 						ERR_error_string(ERR_get_error(), NULL));
 				return -1;
